### PR TITLE
Run rustfmt on all the code

### PR DESCRIPTION
**Pull request by JP-Ellis**
I'm not sure whether this can (and should) be automated with Travis or something?

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -183,7 +182,8 @@ impl Decimal {
 
         let old_scale = self.scale();
 
-        if dp < old_scale && dp < 20 { // Technically, it's 28...
+        if dp < old_scale && dp < 20 {
+            // Technically, it's 28...
```
Comment (paupino) [inline src/lib.rs]: This comment is not great, but loses context here. In fact, I think it's probably a bug which needs another issue created (e.g. why can't I round more than 20 dp?).

File: src/postgres.rs [hunk 2]
```diff
@@ -170,27 +165,42 @@ impl ToSql for Decimal {
...
         } else {
             whole_portion.len() as i16 - 1
         };
-        let all_groups = whole_portion.into_iter().chain(decimal_portion.into_iter())
-            .skip_while(|ref x| *x == "0000").collect::<Vec<String>>();
+        let all_groups = whole_portion
+            .into_iter()
+            .chain(decimal_portion.into_iter())
+            .skip_while(|ref x| *x == "0000")
+            .collect::<Vec<String>>();
```
Comment (paupino) [inline src/postgres.rs]: These are all much cleaner!

**Comment by paupino** [comment]
Will merge however we should address that one comment via an issue (it's limited thinking from what I can determine right now - if it's not, then the comment isn't doing justice).

File: src/lib.rs (hunk 1 above)
Comment (JP-Ellis) [inline src/lib.rs]: The comment doesn't have much context in the first place.  I don't think the change from rustfmt made things any better or worse.

Comment (paupino) [inline src/lib.rs]: Created #23 to address

**Pull request merged**